init.fish

```diff
@@ -0,0 +1,3 @@
+set -xg ANSIBLE_HOSTS (pwd)/hosts
+set -xg ANSIBLE_INVENTORY (pwd)/hosts
+set -xg ANSIBLE_TRANSPORT "ssh"
```

Comment:
I thought that fish used the same syntax than bash to export parameters.

In any case, I'm fine with this commit.